src/kili/adapters/kili_api_gateway/user/operation_mixin.py

```diff
@@ -53,7 +53,7 @@ def create_user(self, data: CreateUserDataKiliGatewayInput, fields: ListOrTuple[
         """Create a user."""
         fragment = fragment_builder(fields)
         query = get_create_user_mutation(fragment)
-        variables = create_user_data_mapper(data)
+        variables = {"data": create_user_data_mapper(data)}
```

Comment:
we prefer put the data here, just a convention. (see update_password method below)

Reply:
Oh ok no problem